devel/addon/ingressnginx/install.sh

```diff
@@ -51,7 +51,7 @@ helm repo update
 helm upgrade \
     --install \
     --wait \
-    --version 3.15.2 \
+    --version 3.31.0 \
```

Comment:
I attempted to configure ingress-nginx for non-root operation, but the containerPort values in the Helm chart don't seem to change the nginx listening port. See https://github.com/helm/charts/issues/14605

So I've configured kyverno to ignore the ingress-nginx namespace.

Reply:
I'll move this change to a separate PR, since it's not necessary here.

Reply:
See https://github.com/jetstack/cert-manager/pull/4046